tests/conftest.py

```diff
@@ -50,7 +51,6 @@ def _database(self):
             conn.execute('CREATE DATABASE {db_name}'.format(db_name=db_name))
 
         dburl = copy.deepcopy(self.engine.url)
-        dburl.database = db_name
```

Comment:
I think removing this line may have caused the builds to interfere with one another. I'm guessing that they're all executing in a default database rather than in the randomly named one.

Reply:
For a way around this, see: https://github.com/sqlalchemy-redshift/sqlalchemy-redshift/compare/sqa-1.4?expand=1#diff-e52e4ddd58b7ef887ab03c04116e676f6280b824ab7469d5d3080e5cba4f2128L53-R57

Reply:
Looks like the workaround is indeed sufficient to get tests passing! https://travis-ci.com/github/sqlalchemy-redshift/sqlalchemy-redshift/builds/231942673

Reply:
That would make sense, I must have accidentally deleted this when I was changing things around to work with my Redshift cluster.

@jklukas added it back, feel free to try integration tests again.

Reply:
The literal line `dburl.database = db_name` fails in 1.4 because URL has become immutable. In 1.4, it's necessary to use a `set` method that returns a new object:

```
         try:
            # SQAlchemy 1.3 and below.
            dburl.database = db_name
        except AttributeError:
            dburl = dburl.set(database=db_name)
```